packages/service-worker/worker/src/driver.ts

```diff
@@ -380,6 +380,10 @@ export class Driver implements Debuggable, UpdateSource {
         }
         break;
       }
+      case 'fetch' {
+        await fetch(urlToOpen);
```

Comment:
This should access `fetch` through `scope` (to make testing easier):

```suggestion
        await this.scope.fetch(urlToOpen);
```